types/index.d.ts

```diff
@@ -1454,7 +1455,7 @@ export interface MessageTimestampProps {
   message?: Client.MessageResponse;
   calendar?: boolean;
   format?: string;
-  tDateTimeParser?(datetime: string | number): Dayjs.Dayjs;
+  tDateTimeParser?: TDateTimeParser;
```

Comment:
I see formatDate below this and it should be abc sorted within these optional types